linode/manager
# test: [M3-8430] - Mock disable OBJ Gen 2 flags for existing OBJ Cypress tests

## Description 📝
Ensures that existing OBJ cypress tests (those not in the ObjectStorageGen2 package) will not have any UI from OBJ gen 2 showing up

## Changes  🔄
- mock gen2 flag as false for obj legacy/multicluster tests
- make sure obj gen2 account capability is not included in mocks

## Target release date 🗓️
n/a

## How to test 🧪
Ensure all tests in the objectStorage package continue to pass
```
yarn cy:run -s "cypress/e2e/core/objectStorage/access-key.e2e.spec.ts"
```
```
yarn cy:run -s "cypress/e2e/core/objectStorage/access-keys.smoke.spec.ts"
```
```
yarn cy:run -s "cypress/e2e/core/objectStorage/enable-object-storage.spec.ts"
```
```
yarn cy:run -s "cypress/e2e/core/objectStorage/object-storage-errors.spec.ts"
```
```
yarn cy:run -s "cypress/e2e/core/objectStorage/object-storage.e2e.spec.ts"
```
```
yarn cy:run -s "cypress/e2e/core/objectStorage/object-storage.smoke.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*
